### ISSUE TITLE
More Ruby 2.7 deprecation warnings

### ISSUE THREAD

**Issue by synthead** [zverok/yard-junk#32]
When running `yard-junk` with Ruby 2.7.1, I get the following deprecation warnings:

```
$ yard-junk
Running YardJunk janitor (version 0.0.8)...

/home/max/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/yard-junk-0.0.8/lib/yard-junk/janitor.rb:42: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/home/max/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/yard-junk-0.0.8/lib/yard-junk/janitor/base_reporter.rb:46: warning: The called method `stats' is defined here

0 failures, 0 problems (0 seconds to run)
```

Duplicate of https://github.com/zverok/yard-junk/issues/30, but I am not able to comment or reopen https://github.com/zverok/yard-junk/issues/30, so I'm filing a new issue here.

**Comment by zverok** [comment]
Ugh, indeed, sorry for that. 0.0.9 should have no warnings, please check (not closing this one in case I still screwed it up)

**Comment by pboling** [comment]
@zverok I've been using this gem for a while now, and I just saw this:
```
...
Methods:         7 (    2 undocumented)
 40.91% documented
rake aborted!
NameError: uninitialized constant YardJunk::Janitor::HtmlReporter::OpenStruct
/Users/pboling/.asdf/installs/ruby/2.7.8/bin/bundle:25:in `load'
/Users/pboling/.asdf/installs/ruby/2.7.8/bin/bundle:25:in `<main>'
Tasks: TOP => default => yard:junk
```

**Comment by pboling** [comment]
It is a deprecation warning starting in Ruby 2.7, and it will be a hard error in Ruby 3.5.  Fix is here: #42

**Comment by zverok** [comment]
Thanks for the fix. Hopefully, v.0.0.10 should be OK for the current and upcoming Ruby.
